l1-contracts/src/UniFiAVSManager.sol

```diff
@@ -386,8 +416,9 @@ contract UniFiAVSManager is UniFiAVSManagerStorage, IUniFiAVSManager, UUPSUpgrad
      * @inheritdoc IUniFiAVSManager
```

Comment:
I think that ++i is cheaper that i++ gas-wise, but it is a minor saving. Idk if the compiler changed that. Not that important tbh, but worth doing a remix test with the latest compiler maybe